modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
```

Comment:
This will be a breaking change, so you'll need to put a major bump in `versions.txt`

Reply:
Bumped in `versions.txt` and in `alloy/package.json` as well [here](https://github.com/tinymce/tinymce/pull/8294/commits/c709ad0dd777e3f481d7ed976384ccd6ca1ad63a).